bomb_planted.py

```diff
@@ -0,0 +1,74 @@
+"""
+It's just another average day in the ER, your fresh out of your Residency and ready to really make a difference in the medical field
+In comes a patient at breakneck speed, police following suit, you ask the officer leading the rest what the situation is.
+
+
+Someone, You have no clue who, you have no clue why
+Has planted an actual fucking bomb in this guy disguised amongst 13 multiple cheap plastic game pieces to the game "Operation"
+
+God Help You
+
+-In this game mode all of the pieces will be randomized
+-All of them will be set to 0 rumble power... Except 1, which will be set to 100%, hitting that piece will make the game rumble for 3 seconds
+-The game will print a list of all the pieces, select the one you will be attempting to remove and then perform the operation
+-Press space when youve finished removing the item and keep going
+- You win by removing all but the "Bomb"
+
+Choose wisely
+"""
+from GamePieces import *
+from picocom import PiPicoCommunicator
+import random
+import keyboard
+import time
+
+
+GamePieceArr = create_game_pieces() #create an array of all the Operation game pieces that inherit from GamePieces
+random.choice(GamePiece).setMotorPower(100) #Randomly select one game piece in the array to plant the "bomb"
```

Comment:
Whoopsies forgot to refer to GamePieceArr here and just referred to the GamePiece class